definitions/region/common.yaml

```diff
@@ -152,59 +153,59 @@
-      description: Centrally-planned Asia, primarily China.
+      description: Centrally-planned Asia, primarily China
       ar6: R10CHINA+
-      countries: China, Cambodia, Hong Kong, Laos, Macao, Mongolia, North Korea,
-        Viet Nam
+      countries: China, Cambodia, Hong Kong, Laos, Macao, Mongolia, Myanmar,
+        North Korea, Viet Nam
   - Europe (R10):
-      description: Europe (including Turkey).
+      description: Europe (including Turkey)
       ar6: R10EUROPE
       countries: Austria, Belgium, Bosnia and Herzegovina, Bulgaria, Croatia, Cyprus,
         Czech Republic, Denmark, Estonia, France, Finland, Germany, Greece, Hungary,
         Iceland, Ireland, Italy, Latvia, Lithuania, Luxembourg, Malta, Montenegro,
         Netherlands, North Macedonia, Norway, Portugal, Switzerland, Poland, Romania,
         Serbia, Slovakia, Slovenia, Spain, Sweden, Turkey, United Kingdom
   - India+ (R10):
-      description: South Asia, primarily India.
+      description: South Asia, primarily India
       ar6: R10INDIA+
       countries: India, Afghanistan, Bangladesh, Bhutan, Maldives, Nepal, Pakistan,
         Sri Lanka
   - Latin America (R10):
-      description: Latin America and the Caribbean.
+      description: Latin America and the Caribbean
       ar6: R10LATIN_AM
       countries: Argentina, Bahamas, Barbados, Belize, Bolivia, Brazil, Chile, Colombia,
         Costa Rica, Cuba, Dominican Republic, Ecuador, El Salvador, Grenada, Guadeloupe,
         Guatemala, Guyana, Haiti, Honduras, Jamaica, Martinique, Mexico, Nicaragua,
         Panama, Paraguay, Peru, Puerto Rico, Saint Lucia, Suriname, Trinidad and Tobago,
         Uruguay, Venezuela
   - Middle East (R10):
-      description: Middle East.
+      description: Middle East
       ar6: R10MIDDLE_EAST
       countries: Bahrain, Iraq, Iran, Israel, Jordan, Kuwait, Lebanon, Oman, Qatar,
         Saudi Arabia, Syria, United Arab Emirates, Yemen
       note: Depending on the spatial resolution, some models report Northern Africa
         as part of the Middle East region.
   - North America (R10):
-      description: United States and Canada.
+      description: United States and Canada
       ar6: R10NORTH_AM
       countries: Canada, Guam, United States
   - Pacific OECD (R10):
-      description: Pacific OECD.
+      description: Pacific OECD (membership status 1990)
       ar6: R10PAC_OECD
       countries: Australia, Japan, New Caledonia, New Zealand, Samoa, Solomon Islands,
         Vanuatu
   - Reforming Economies (R10):
-      description: Former Soviet Union.
+      description: Former Soviet Union
       ar6: R10REF_ECON
       countries: Armenia, Azerbaijan, Belarus, Georgia, Kazakhstan, Kyrgyzstan, Moldova,
         Russian Federation, Tajikistan, Turkmenistan, Ukraine, Uzbekistan
   - Rest of Asia (R10):
-      description: Other countries of Asia not included in China+ or India+.
+      description: Other countries of Asia not included in China+ or India+
```

Comment:
description: Other countries of Asia not included in China+, India+ or Reforming Economies